src/utils/aeternity.js

```diff
@@ -95,8 +96,13 @@ aeternity.initClient = async () => {
     try {
       aeternity.client = await aeternity.initStaticClient();
       result = await aeternity.initProvider();
-    } catch (e) {
-      console.error(e);
+    } catch (err) {
+      router.push({
```

Comment:
i think this redirect could become annoying for users. Imagine typing something and you get an error, you will lose what you were doing. I think might be better to show some popup or similar, so user could keep their position (or whatever they do).

Reply:
Ok, but what is the point of new static page then?

Reply:
i don't actually see a point, but could be a temporary solution for the end-users for now